core/deployment/src/main/java/io/quarkiverse/langchain4j/deployment/config/LangChain4jBuildConfig.java

```diff
@@ -36,6 +36,12 @@ public interface LangChain4jBuildConfig {
      */
     DevServicesConfig devservices();
 
+    /**
+     * Whether the response schema can be used in the prompt
```

Comment:
This description isn't very clear to me - should it say 
```
the {response_schema} placeholder
```
instead of just
```
response schema
```
?

Reply:
maybe repeat here what you said in https://github.com/quarkiverse/quarkus-langchain4j/issues/679#issuecomment-2168722583 :)

Reply:
Ok, I can think of way to improve this comment.